packages/dnb-eufemia/src/components/upload/types.ts

```diff
@@ -0,0 +1,55 @@
+import { SkeletonShow } from '../skeleton/Skeleton'
+
+export interface UploadProps {
+  /**
+   * unique id used with the useUpload hook to manage the files
+   */
+  id: string
+
+  /**
+   * list of accepted file types.
+   */
+  acceptedFileTypes: string[]
+
+  /**
+   * Custom className on the component root
+   * Default: null
+   */
+  className?: string
+
+  /**
+   * Skeleton should be applied when loading content
+   * Default: null
+   */
+  skeleton?: SkeletonShow
+
+  /**
+   * If set true, accepting multiple files is allowed
+   */
+  multipleFiles?: boolean
+
+  /**
+   * fileMaxSize is max size of each file in MB
+   */
+  fileMaxSize?: number
+
+  /**
+   * Custom text properties
+   */
+  title?: React.ReactNode
+  text?: React.ReactNode
+  formatsDescription?: React.ReactNode
+  fileSizeDescription?: React.ReactNode
+  fileSizeContent?: React.ReactNode
+  uploadButtonText?: React.ReactNode
+  uploadingLoadingText?: React.ReactNode
```

Comment:
```suggestion
  uploadLoadingText?: React.ReactNode
```

Could/should we use the same text pattern as `uploadButtonText`?
Or is it intentional that we prefix with the word `upload` for one of the props, and `uploading` for the other one?